Lib/Src/CDispatcher.cpp

```diff
@@ -203,16 +215,14 @@ void CDispatcher::processComChannels()
                             mp_comchannels[channel]);
                     controller++;
                 }
-            }
-            if (!b_command_recognised)
-            {
-                etl::string<MAX_STRING_SIZE> message;
-                message = "COMMAND_NOT_RECOGNISED\n";
-                mp_comchannels[channel]->send(message);
+                if (!b_command_recognised)
+                {
+                    mp_comchannels[channel]->send(COMMAND_NOT_RECOGNISED);
```

Comment:
It would be useful to send a short message like COMMAND_OK, COMMAND_NOT_RECOGNISED, COMMAND_MALFORMAT for testing.